lib/vectortilefeature.js

```diff
@@ -192,6 +197,57 @@ VectorTileFeature.prototype.toGeoJSON = function(x, y, z) {
     return result;
 };
 
+VectorTileFeature.prototype.toOffsetsJSON = function() {
+    // Resize tiles to 512x512 bounding box
+    var tileSize = 512;
+    var tileExtent = this.extent;
+    var transformFunction = function (x, y) {
+        return [
+            (x/tileExtent) * tileSize,
+            (y/tileExtent) * tileSize
+        ];
+    };
+
+    var type = VectorTileFeature.types[this.type];
+    var geometryOffsets = this.loadGeometry(transformFunction);
+    var i;
+
+    switch (this.type) {
+        case 1:
```

Comment:
don't they have an enum-like on type?

Reply:
I wish they had one 😓. Given that they have an array with those types I can do something like:

```js
var type = VectorTileFeature.types[this.type];

switch (type) {
    case 'Point':
        break;
}
```

but I guess int comparison is faster 😁

Reply:
maybe we can add a little comment with the geometry type...